MANIFEST.in

```diff
@@ -20,6 +20,7 @@ prune docs/_build
 recursive-include docs *.png
 recursive-include docs *.py
 recursive-include docs *.rst
```

Comment:
Since we don't have anymore `*.rst` files, we could remove the following two lines from `MANIFEST.in`:

```
include *.rst
recursive-include docs *.rst
```